x-pack/plugins/synthetics/public/apps/synthetics/hooks/use_synthetics_priviliges.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useSelector } from 'react-redux';
+import React from 'react';
+import {
+  EuiEmptyPrompt,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiMarkdownFormat,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { css } from '@emotion/react';
+import { i18n } from '@kbn/i18n';
+import { selectOverviewStatus } from '../state/overview_status';
+import { SYNTHETICS_INDEX_PATTERN } from '../../../../common/constants';
+
+export const useSyntheticsPrivileges = () => {
```

Comment:
might be nice to (also) have this as a simple hook which returns an object or boolean rather than a component, for other use cases in the future. No big deal though.